README.md

```diff
@@ -34,15 +38,15 @@ More detailed instructions are on our ["Usage"](/docs/USAGE.md) page.
 # Run the app using Docker
 ### Setup
 1. Download and install [Docker](https://docs.docker.com/get-started/)
-1. Pull the [latest Docker image for this project](https://github.com/orgs/MobilityData/packages/container/package/gtfs-validator). For example, `docker pull ghcr.io/mobilitydata/gtfs-validator:v2.0.0`.
+1. Pull the [latest Docker image for this project](https://github.com/orgs/MobilityData/packages/container/package/gtfs-validator). For example, `docker pull ghcr.io/mobilitydata/gtfs-validator:fix-debian-version-used-in-docker-image`.
 
 ### Run it
 
 #### For Mac and Linux
 
 To run the Docker image in a new container:
 
-`docker run -v /myDirectory:/theContainerDirectory -it ghcr.io/mobilitydata/gtfs-validator:v2.0.0`
+`docker run -v /myDirectory:/theContainerDirectory -it ghcr.io/mobilitydata/gtfs-validator:snapshot`
```

Comment:
Will `snapshot` always point to the latest docker image? If so, can we use that above in "Setup" section too?

Reply:
https://github.com/MobilityData/gtfs-validator/pull/913 provides support to publish a new docker image using the `latest` tag which will allow us to automatically point to it in the documentation. @barbeau  PTAL

> Will snapshot always point to the latest docker image? If so, can we use that above in "Setup" section too?

After the above is reviewed and merge to master - yes.

Reply:
Also, it will be "latest" instead of "snapshot".